examples/with-firebase-authentication/pages/index.js

```diff
@@ -51,7 +51,7 @@ const Index = () => {
         </Link>
       </div>
       {error && <div>Failed to fetch food!</div>}
-      {data ? (
+      {data && !error ? (
```

Comment:
This was a tiny UI bug - when an error happened it still showed "Loading..." as well as  "Failed to fetch food!"